test/openjd/cli/test_run_command.py

```diff
@@ -295,7 +471,10 @@ def test_do_run_success(
         )
 
         # WHEN
-        do_run(args)
+        try:
+            do_run(args)
+        except:  # noqa: E722
+            pass
 
         # THEN
```

Comment:
The assertions in this test look pretty light, especially with the try-pass. Can we check that the exit actions are run and in the correct order? It could be easier to make a new unit test instead for the order assertions.

Reply:
Yeah, the ordering of actions is verified by the regex that it's matching against. For example, [this](https://github.com/OpenJobDescription/openjd-cli/blob/e5ea7d5e29ca08d8b448c06723487cbf357b1b93/test/openjd/cli/test_run_command.py#L245) makes sure that the print statements from each action appears in the expected order in the log.